### ISSUE TITLE
simplify.py: error: unrecognized arguments: Notebooks/muss/resources/models/muss_fr_mined/model.pt

### ISSUE THREAD

**Issue by oskrmiguel** [facebookresearch/muss#26]
Hello when I run this command 

!python '/content/drive/MyDrive/Colab Notebooks/muss/scripts/simplify.py' '/content/drive/MyDrive/Colab Notebooks/muss/scripts/examples.fr' --model-name muss_fr_mined

in Google Colab I get this error

INFO:root:Generating grammar tables from /usr/lib/python3.7/lib2to3/Grammar.txt
INFO:root:Generating grammar tables from /usr/lib/python3.7/lib2to3/PatternGrammar.txt
Downloading...
... 100% - 6204 MB - 10.12 MB/s - 612s
Extracting...
usage: simplify.py [-h] [--no-progress-bar] [--log-interval LOG_INTERVAL]
                   [--log-format {json,none,simple,tqdm}]
                   [--tensorboard-logdir TENSORBOARD_LOGDIR] [--seed SEED]
                   [--cpu] [--tpu] [--bf16] [--memory-efficient-bf16] [--fp16]
                   [--memory-efficient-fp16] [--fp16-no-flatten-grads]
                   [--fp16-init-scale FP16_INIT_SCALE]
                   [--fp16-scale-window FP16_SCALE_WINDOW]
                   [--fp16-scale-tolerance FP16_SCALE_TOLERANCE]
                   [--min-loss-scale MIN_LOSS_SCALE]
                   [--threshold-loss-scale THRESHOLD_LOSS_SCALE]
                   [--user-dir USER_DIR] [--empty-cache-freq EMPTY_CACHE_FREQ]
                   [--all-gather-list-size ALL_GATHER_LIST_SIZE]
                   [--model-parallel-size MODEL_PARALLEL_SIZE]
                   [--checkpoint-suffix CHECKPOINT_SUFFIX]
                   [--checkpoint-shard-count CHECKPOINT_SHARD_COUNT]
                   [--quantization-config-path QUANTIZATION_CONFIG_PATH]
                   [--profile]
                   [--criterion {wav2vec,label_smoothed_cross_entropy,label_smoothed_cross_entropy_with_alignment,adaptive_loss,nat_loss,sentence_ranking,legacy_masked_lm_loss,composite_loss,cross_entropy,ctc,sentence_prediction,masked_lm,vocab_parallel_cross_entropy}]
                   [--tokenizer {nltk,moses,space}]
                   [--bpe {gpt2,subword_nmt,hf_byte_bpe,fastbpe,sentencepiece,characters,bert,bytes,byte_bpe}]
                   [--optimizer {adagrad,adafactor,sgd,lamb,adamax,adadelta,nag,adam}]
                   [--lr-scheduler {fixed,reduce_lr_on_plateau,triangular,cosine,tri_stage,polynomial_decay,inverse_sqrt}]
                   [--scoring {wer,chrf,sacrebleu,bleu}] [--task TASK]
                   [--num-workers NUM_WORKERS]
                   [--skip-invalid-size-inputs-valid-test]
                   [--max-tokens MAX_TOKENS] [--batch-size BATCH_SIZE]
                   [--required-batch-size-multiple REQUIRED_BATCH_SIZE_MULTIPLE]
                   [--required-seq-len-multiple REQUIRED_SEQ_LEN_MULTIPLE]
                   [--dataset-impl {raw,lazy,cached,mmap,fasta}]
                   [--data-buffer-size DATA_BUFFER_SIZE]
                   [--train-subset TRAIN_SUBSET] [--valid-subset VALID_SUBSET]
                   [--validate-interval VALIDATE_INTERVAL]
                   [--validate-interval-updates VALIDATE_INTERVAL_UPDATES]
                   [--validate-after-updates VALIDATE_AFTER_UPDATES]
                   [--fixed-validation-seed FIXED_VALIDATION_SEED]
                   [--disable-validation]
                   [--max-tokens-valid MAX_TOKENS_VALID]
                   [--batch-size-valid BATCH_SIZE_VALID]
                   [--curriculum CURRICULUM] [--gen-subset GEN_SUBSET]
                   [--num-shards NUM_SHARDS] [--shard-id SHARD_ID]
                   [--distributed-world-size DISTRIBUTED_WORLD_SIZE]
                   [--distributed-rank DISTRIBUTED_RANK]
                   [--distributed-backend DISTRIBUTED_BACKEND]
                   [--distributed-init-method DISTRIBUTED_INIT_METHOD]
                   [--distributed-port DISTRIBUTED_PORT]
                   [--device-id DEVICE_ID] [--distributed-no-spawn]
                   [--ddp-backend {c10d,no_c10d}]
                   [--bucket-cap-mb BUCKET_CAP_MB] [--fix-batches-to-gpus]
                   [--find-unused-parameters] [--fast-stat-sync]
                   [--broadcast-buffers] [--distributed-wrapper {DDP,SlowMo}]
                   [--slowmo-momentum SLOWMO_MOMENTUM]
                   [--slowmo-algorithm SLOWMO_ALGORITHM]
                   [--localsgd-frequency LOCALSGD_FREQUENCY]
                   [--nprocs-per-node NPROCS_PER_NODE]
                   [--pipeline-model-parallel]
                   [--pipeline-balance PIPELINE_BALANCE]
                   [--pipeline-devices PIPELINE_DEVICES]
                   [--pipeline-chunks PIPELINE_CHUNKS]
                   [--pipeline-encoder-balance PIPELINE_ENCODER_BALANCE]
                   [--pipeline-encoder-devices PIPELINE_ENCODER_DEVICES]
                   [--pipeline-decoder-balance PIPELINE_DECODER_BALANCE]
                   [--pipeline-decoder-devices PIPELINE_DECODER_DEVICES]
                   [--pipeline-checkpoint {always,never,except_last}]
                   [--zero-sharding {none,os}] [--path PATH]
                   [--remove-bpe [REMOVE_BPE]] [--quiet]
                   [--model-overrides MODEL_OVERRIDES]
                   [--results-path RESULTS_PATH] [--beam N] [--nbest N]
                   [--max-len-a N] [--max-len-b N] [--min-len N]
                   [--match-source-len] [--no-early-stop] [--unnormalized]
                   [--no-beamable-mm] [--lenpen LENPEN] [--unkpen UNKPEN]
                   [--replace-unk [REPLACE_UNK]] [--sacrebleu]
                   [--score-reference] [--prefix-size PS]
                   [--no-repeat-ngram-size N] [--sampling]
                   [--sampling-topk PS] [--sampling-topp PS]
                   [--constraints [{ordered,unordered}]] [--temperature N]
                   [--diverse-beam-groups N] [--diverse-beam-strength N]
                   [--diversity-rate N] [--print-alignment] [--print-step]
                   [--lm-path PATH] [--lm-weight N]
                   [--iter-decode-eos-penalty N] [--iter-decode-max-iter N]
                   [--iter-decode-force-max-iter] [--iter-decode-with-beam N]
                   [--iter-decode-with-external-reranker]
                   [--retain-iter-history] [--retain-dropout]
                   [--retain-dropout-modules RETAIN_DROPOUT_MODULES [RETAIN_DROPOUT_MODULES ...]]
                   [--decoding-format {unigram,ensemble,vote,dp,bs}]
                   [--force-anneal N] [--lr-shrink LS] [--warmup-updates N]
                   [-s SRC] [-t TARGET] [--load-alignments]
                   [--left-pad-source BOOL] [--left-pad-target BOOL]
                   [--max-source-positions N] [--max-target-positions N]
                   [--upsample-primary UPSAMPLE_PRIMARY] [--truncate-source]
                   [--num-batch-buckets N] [--eval-bleu]
                   [--eval-bleu-detok EVAL_BLEU_DETOK]
                   [--eval-bleu-detok-args JSON] [--eval-tokenized-bleu]
                   [--eval-bleu-remove-bpe [EVAL_BLEU_REMOVE_BPE]]
                   [--eval-bleu-args JSON] [--eval-bleu-print-samples] --langs
                   LANG [--prepend-bos]
                   data
simplify.py: error: unrecognized arguments: Notebooks/muss/resources/models/muss_fr_mined/model.pt

**Comment by louismartin** [comment]
Which version of fairseq do you have installed?
`!pip freeze | grep fairse` or `import fairseq; print(fairseq.__version__)`

**Comment by oskrmiguel** [comment]
fairseq==0.10.2
0.10.2

**Comment by sadakmed** [comment]
Hi @louismartin  when I was checking the resources it was created under `site-packages` and not under muss, can you confirm?

**Comment by louismartin** [comment]
Hi @sadakmed, thanks for the question, it's better to open a new issue for your question, head over to #29
